server/Arcadia.Assistant.Calendar/Arcadia.Assistant.Calendar.csproj

```diff
@@ -6,7 +6,10 @@
 
   <ItemGroup>
     <PackageReference Include="Akka" Version="1.3.4" />
+    <PackageReference Include="Akka.DI.Core" Version="1.3.4" />
     <PackageReference Include="Akka.Persistence" Version="1.3.4" />
+    <PackageReference Include="MailKit" Version="2.0.3" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.0.0" />
```

Comment:
Having Configuraiton library linked here is not right, interface should be used instead of IConfigurationSection